src/common/Channel.cpp

```diff
@@ -278,6 +278,13 @@ bool Channel::canSendMessage() const
     return false;
 }
 
+bool Channel::isWritable() const
+{
+    using Type = Channel::Type;
+    auto type = this->getType();
+    return type != Type::TwitchMentions && type != Type::TwitchLive;
```

Comment:
Why `TwitchMentions` in specific? What about `TwitchLive` or TwitchWhispers`? seems like some weird "magic" comparison to make.

Reply:
You can use the split input in the whispers split for `/r`/`/w` usage) as well as in the `TwitchWatching` channel